src/data/faq_de.json

```diff
@@ -438,6 +438,16 @@
                             "Nein, das ist leider nicht möglich. Jedes positive Test-Ergebnis, das in die App geladen wird, muss zuvor von Laboren überprüft worden sein. Diese stellen sicher, dass kein Missbrauch stattfindet und keine falsche Ergebnisse eingetragen werden. Dazu sind zur Zeit nur Labore in Deutschland angeschlossen. Sie können also kein positives Testergebnis aus anderen Ländern, zum Beispiel aus den Niederlanden, eingeben."
                         ]
                     },
+                    {
+                        "title": "Kann die Corona-Warn-App die Kranschreibung vom Arzt ersetzen?",
+                        "anchor": "sick_leave",
+                        "active": true,
+                        "textblock": [
+                            "Nein, das kann sie nicht. Das Verfahren des QR-Codes wurde zur Verifizierung von positiven Testergebnissen entwickelt. Die Information über die Corona-Warn-App ersetzt nicht den Befundbrief bzw. die Information vom Arzt oder Labor. Nur sie können Ihnen auch einen Nachweis für Ihren Arbeitgeber geben. ",
+                            "Der Grund ist folgender: Weil die Corona-Warn-App den dezentralen Ansatz verfolgt, enthalten die Benachrichtigungen der Corona-Warn-App keine persönlichen Informationen wie Name oder Geburtsdatum. Diese Identifizierungsmerkmale sind jedoch für eine Befreiung aus der Quarantäne notwendig.",
+                            "Zusätzlich zur Anzeige Ihres Ergebnisses in der Corona-Warn-App sollten Sie immer eine schriftliche Nachricht durch das Labor, die Teststelle oder Ihre Arztpraxis bekommen."
```

Comment:
Die Fragestellungen im App Store sind meist eher in die andere Richtung, also nicht Krankschreibung sondern Entlassung aus der Quarantäne, wenn man ein negatives Testergebnis hat. Daher würde ich dafür entweder eine zusätzliche FAQ machen oder das hier im Titel mit einbringen. 
Wobei ich mir nicht sicher bin, dass wir das hier in unsere FAQ reinbringen sollten. Das ist für mich ein RKI-Thema und die haben es doch jetzt auch in ihrer FAQ?

Reply:
ok